<launch>

<arg name="axis_lin" default="1"/>
<arg name="axis_ang" default="0"/>
<arg name="linear" default="0.03"/>
<arg name="angular" default="0.03"/>
<arg name="js_dev" default="/dev/input/js0"/>
<arg name="deadzone" default="0.12"/>
    <node  pkg="joy" type="joy_node" name="joystick" output="screen">
        <param name="dev" value="$(arg js_dev)" type="string"/>
        <param name="deadzone" value="$(arg deadzone)" />
    </node>
    <node pkg="my_joy" type="joy_control" name="joy_control" output="screen">
        <param name="axis_linear" type="int" value="$(arg axis_lin)"/>
        <param name="axis_angular" type="int" value="$(arg axis_ang)"/>
        <param name="vel_linear" type="double" value="$(arg linear)"/>
        <param name="vel_angular" type="double" value="$(arg angular)"/>
    </node>
    
</launch>
